Cargo.toml

```diff
@@ -2,10 +2,9 @@
 resolver = "2"
 members = [
     "auto-deref",
-    "client",
-    "server",
+    "app",
     "chess",
     "common",
```

Comment:
I think `common` and `build_common` could also be modules in `app`, but this can easily be a separate PR.

Reply:
yeah Ill make an issue